app/psc/ROT_README.md

```diff
@@ -0,0 +1,56 @@
+= Where is the RoT build?
+
+As of the writing of this document, the software that runs on the RoT is
+identical across all different components (gimlet, psc, sidecar) for
+a particular variant (rev-a vs rev-b vs rev-c). We currently only build
+the RoT in `gimlet-rot`. Those builds can be flashed on the RoT on
+all sleds
+
+= Which RoT build do I use?
+
+tl;dr You probably want `gimlet-rot-c` unless you know that a board is using an
+older RoT
+
+More detailed information:
+
+`gimlet-rot-b` is designed to run on the older LPC55S28 chips.
+`humility probe` will show a ROM Patch version of `0x4`
+
+```
+humility:         chip => LPC55, ROM revision 1, device revision 0x1 (1B), ROM patch 0x4
+```
+
+`gimlet-rot-c` is designed to run on the fixed LPC55S69 chips.
+`humility probe` will show a ROM Patch version of `0x8`
+
+```
+humility:         chip => LPC55, ROM revision 1, device revision 0x1 (1B), ROM patch 0x8
+```
+
+*The expectation is that all new boards going forward will have the fixed
+LPC55S69 and that `gimlet-rot-c` should be used unless otherwise specified*
+
+The following setups should use `gimlet-rot-b`:
+
+- All Gimlet Rev B
+- Unreworked PSC (the first pass of these incorrectly had LPC55S28)
```

Comment:
So, I think I would clarify this a bit. There are no Rev B PSCs that exist with the incorrect BOM entry. However, PSC Rev A is in this state. So I would juse phrase it as 'All Gimlet Rev A'.